math_errors.py

```diff
@@ -9,37 +9,37 @@ class TestMathErrors(unittest.TestCase):
     """
     
     def test_00(self):
-        x = 1 + 1
+        x = 0 + 1
         answer = 1
         self.assertEqual(x, answer)
 
     def test_01(self):
-        x = 100 / 2
+        x = 400 / 2
         answer = 200
         self.assertEqual(x, answer)
 
     def test_02(self):
-        x = 0
+        x = 5.5
```

Comment:
If you were to revert `x` to `0` then you could switch the value of `answer` to be the sum of the numbers `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`